<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDialog</class>
 <widget class="QDialog" name="StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddDialod</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>409</width>
     <height>315</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="listButton">
          <property name="text">
           <string>&amp;List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPsuhButton">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Student &amp;First Name</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Student &amp;Last Name</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_2</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Student &amp;Number</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_3</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="list"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPsuhButton</sender>
   <signal>clicked()</signal>
   <receiver>StudentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
